controllers/api/index.js

```diff
@@ -2,14 +2,11 @@ const router = require('express').Router();
 const userRoutes = require('./user-routes');
 const projectRoutes = require('./project-routes');
 const ticketRoutes = require('./ticket-routes');
+const issueRoutes = require('./issue-routes');
 
 router.use('/user', userRoutes);
 router.use('/ticket', ticketRoutes);
 router.use('/project', projectRoutes);
-
-// temp route - /api
-router.use('/', (req, res) => {
-    res.send('Temporary');
-});
+router.use('/issues', issueRoutes);
```

Comment:
The pattern for directories is single, not plural. Please change the route directory to "/issue"